src/queries/models/user/getUserSubscriptionsModelCreator.ts

```diff
@@ -0,0 +1,4 @@
+import { UserSubscriptionsResponse } from 'queries/types';
+
+export const getUserSubscriptionsModelCreator = (response: UserSubscriptionsResponse): UserSubscriptionsResponse =>
```

Comment:
@mishramonalisha76 if we are not remodelling the data then why do we have this function and file?